client/webserver/site/src/js/order.ts

```diff
@@ -649,5 +649,9 @@ export const CoinExplorers: Record<number, Record<number, (cid: string) => strin
   145: { // bch
     [Mainnet]: (cid: string) => `https://bch.loping.net/tx/${cid.split(':')[0]}`,
     [Testnet]: (cid: string) => `https://tbch4.loping.net/tx/${cid.split(':')[0]}`
+  },
+  966: { // matic
+    [Mainnet]: (cid: string) => `https://polygonscan.com/tx/${cid.split(':')[0]}`,
+    [Testnet]: (cid: string) => `https://mumbai.polygonscan.com/tx/${cid.split(':')[0]}`
```

Comment:
I think we'll need to do like `ethExplorers` to handle the variable coin id encoding. That should still apply to matic.

Also, where did all the extraneous formatting changes come from?

Reply:
> Also, where did all the extraneous formatting changes come from?

Missed that, I was surprised too.

Reply:
> I think we'll need to do like ethExplorers to handle the variable coin id encoding.

Ah, just noticed we are doing something special for eth. np.